kerchunk/intro.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Kerchunk"
+subtitle: References for (optionally) chunked, compressed n-dimensional arrays
+---
+
+## What is Kerchunk?
+
+Kerchunk is a python library for creating reference files to support cloud-optimized access to traditional geospatial file formats, like NetCDF. Kerchunk negates the need to create and store copies of data for cloud-optimized access. Given the challenge of creating and maintaining copies of data, Kerchunk is a great tool.
+
+::: {.callout-warning}
+A major limitation of kerchunk is the chunking of data will always be constrained to the chunk structure of the underlying data format.
```

Comment:
Do we have a page explaining what a Chunk is? people from HDF5 background might not know since only Zarr and Kerchunk really talk about it that way (rasters call them internal tiles)